### ISSUE TITLE
Alpha Weapon crash?

### ISSUE THREAD

**Issue by SPXIII** [Joe-Degler/Lion-Sora-OpenKH#1]
Tried playing a rando seed with this mod. Having Alpha Weapon in the left hand (as is default on Master Form) works fine, but unpausing after equipping Alpha Weapon in the right hand (base Sora) immediately crashes the game. Also worth noting, there was some kind of wire frame showing on the characters in the pause menu, like a layer just over the actual models.

**Comment by Joe-Degler** [comment]
Yeah, I've had a bug report on Nexus as well that some rando keyblades crash the game. For what its worth, all vanilla weapons seem to work, so I assume some of the modded in Keyblades from Rando have a higher filesize. I'll try to reduce the model size further to accomodate those.

Never had that wire frame thing! That's odd. Can you send a screenshot of that?
